app/helpers/rubygems_helper.rb

```diff
@@ -139,6 +139,19 @@ def links_to_owners_without_mfa(rubygem)
     rubygem.owners.without_mfa.sort_by(&:id).inject("") { |link, owner| link << link_to_user(owner) }.html_safe
   end
 
+  def link_to_owner(owner, show_mfa_status: false)
+    link_text = avatar(32, "gravatar-#{owner.id}", owner, class: "tw-rounded-full tw-mr-1")
+    link_text << " #{owner.display_handle}"
+    link_text << " *" if show_mfa_status && owner.mfa_disabled?
+    link_to(
+      link_text.html_safe,
```

Comment:
let's put the link text in a localized html string?

Reply:
There's no actual text so this would just be:

```yaml
owner_link_html: "%{avatar} %{handle}%{mfa_unable}"
```

This is just string interpolation so we should avoid putting a lot of distance between caller and the format, imho.

Reply:
wouldn't the order still be different in an RTL language?

Reply:
I'd like to avoid overthinking it for now. I'm not sure that star placement is going to ruin a RTL language. We don't have any yet anyway, so that will be its own large undertaking, of which this will be a minor detail.